### PR TITLE
Updated development section of readme and about:nightly

### PR DESCRIPTION
**Pull request by UtiluMark**
* Changed the text about where bugs and feature requests are filed in the readme file, fixing #181
* Changed the link to the bug list on bugzilla.mozilla.org to the issues list of the project at GitHub in the readme file, fixing #181
* Changed the link to file a bug on bugzilla.mozilla.org to file new issue of the project at GitHub in the readme file, fixing #181
* Removed link to non-existing forum topic from about:nightly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,10 +35,10 @@ To build for release (no build number in filename):
     ant release
 
 # Development
-All bugs are reported to the Nightly Tester Tools component at bugzilla.mozilla.org. [bug list](https://bugzilla.mozilla.org/buglist.cgi?query_format=advanced&component=Nightly%20Tester%20Tools&product=Other%20Applications), [file a bug](https://bugzilla.mozilla.org/enter_bug.cgi?product=Other%20Applications&component=Nightly%20Tester%20Tools). Check out [the wiki](https://wiki.mozilla.org/Auto-tools/Projects/NightlyTesterTools) for a list of current and proposed features and feel free to file bugs and submit patches.
+All bugs and feature requests are filed in the Nightly Tester Tools project at GitHub. [issues list](https://github.com/mozilla/nightlytt/issues), [file new issue](https://github.com/mozilla/nightlytt/issues/new). Check out [the wiki](https://wiki.mozilla.org/Auto-tools/Projects/NightlyTesterTools) for a list of current and proposed features and feel free to file bugs and submit patches.
```
Comment (whimboo) [inline README.md]: I would rephrase the second sentence like that: "You can find a [list of open issues](..), or you can [file a new issue](..)."

File: extension/chrome/content/aboutNightly/aboutNightly.xhtml [hunk 2]
```diff
@@ -63,28 +63,22 @@
     <ul>
       <li>
         <a href="https://addons.mozilla.org/en-US/firefox/addon/nightly-tester-tools/">
-          addons.mozilla.org page
+          Nightly Tester Tools addons.mozilla.org page
```
Comment (whimboo) [inline extension/chrome/content/aboutNightly/aboutNightly.xhtml]: I don't see why we have to use this prefix. It should be clear that all this is about Nightly tester tools.

**Comment by whimboo** [comment]
Also please correctly rebase against latest master for this PR. There are a lot of unrelated commits. Thanks.

File: README.md (hunk 1 above)
Comment (UtiluMark) [inline README.md]: Rephrased to: "You can [view the list of open issues](https://github.com/mozilla/nightlytt/issues), or you can [file a new issue](https://github.com/mozilla/nightlytt/issues/new)."

File: extension/chrome/content/aboutNightly/aboutNightly.xhtml (hunk 2 above)
Comment (UtiluMark) [inline extension/chrome/content/aboutNightly/aboutNightly.xhtml]: I've reverted adding the prefixes.

**Comment by UtiluMark** [comment]
@whimboo Rebased against latest master, so there is now only one commit.

**Comment by whimboo** [comment]
Looks fine to me now. I will get it merged in a moment. Thanks!
